src/test/java/com/marklogic/spark/MarkLogicSparkWriteDriver.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.marklogic.spark;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.DatabaseClientFactory;
+import com.marklogic.client.document.TextDocumentManager;
+import com.marklogic.spark.writer.MarkLogicWriteDataSource;
+import org.apache.spark.sql.*;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+public class MarkLogicSparkWriteDriver {
```

Comment:
We already have test plumbing to reuse, so let's make use of that - make this into a `WriteRowsTest` that extends `AbstractIntegrationTest`. Can use `newSparkSession()` to create a Spark session and use that to read from the CSV. And then write the results to MarkLogic in a hardcoded collection, and then verify that the collection has N documents in it.

To make the test repeatable, modify `deleteDocumentsBeforeTestRuns` in `AbstractIntegrationTest` to delete the hardcoded collection that this test uses - e.g. 

```
public void deleteDocumentsBeforeTestRuns() {
    getDatabaseClient().newServerEval().javascript("xdmp.collectionDelete('my-test-data')").evalAs(String.class);
}
```